ci/prow/make_config.go

```diff
@@ -857,12 +858,12 @@ func generatePeriodic(title string, repoName string, periodicConfig yaml.MapSlic
 				"--github-token /etc/hub-token/token"}
 			addVolumeToJob(&data.Base, "/etc/hub-token", "hub-token", true)
 			data.Base.Timeout = 90
-		case "performance":
+		case "performance", "performance-mesh":
```

Comment:
Reuse "performance" and just override the args. Because the name will conflict, create a generic key for jobs called "job-name-suffix" and use it to generate the job name.

Next time you create a job like this you won't need any code changes.